app/services/whats_app_adapter/three_sixty_dialog/token_fetcher_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module WhatsAppAdapter
+  module ThreeSixtyDialog
+    class TokenFetcherService
+      def initialize; end
+
+      def call
```

Comment:
```rb
      def self.call
```

For example. Feel free to use a name other than `call.`

Reply:
The reason I had explained that this is a common pattern to name it call is because the service object should only handle one responsibility and the class should describe what it does, not the method.